server/src/frameworks/data-services/postgres/postgres-data-services.module.ts

```diff
@@ -0,0 +1,22 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+import { PostgresDataServices } from './postgres-data-services.service';
+import { ProductEntity } from './entities/product.entity';
+import { IDataServices } from '../../../core/abstracts/data-services.abstract';
```

Comment:
better not to use ../..
use absolute imports from root/src
if you will move some folders\code now it will broke. If you will use imports from root - then no

Reply:
I have some troubles with deploying aliases in code. In the future, I will remove these dependencies.

Reply:
It could be done without aliases, just not forget to add aliases to TSConfig and all will work